static/src/javascripts/projects/commercial/modules/check-dispatcher.js

```diff
@@ -2,15 +2,11 @@
 import config from 'lib/config';
 import { allEmailCanRun, listCanRun } from 'common/modules/email/run-checks';
 import { getListConfigs } from 'common/modules/email/email-article';
-import {
-    emailTests,
-    contributionsTests,
-    userIsInAClashingAbTest,
-} from 'common/modules/experiments/ab-test-clash';
 import { trackAdRender } from 'commercial/modules/dfp/track-ad-render';
 import { commercialFeatures } from 'common/modules/commercial/commercial-features';
 import { checks } from 'common/modules/check-mediator-checks';
 import { resolveCheck, waitForCheck } from 'common/modules/check-mediator';
+import { getTest as getAcquisitionTest } from 'common/modules/experiments/acquisition-test-selector';
```

Comment:
Out of interest why do we have this file and the identical `static/src/javascripts/projects/commercial-control/modules/check-dispatcher.js` file?

Reply:
`commercial-control` is run on a 1% control group to measure the difference between today's commercial system and the "start of the quarter code". See this PR for more info: https://github.com/guardian/frontend/pull/18677 . And here's how we are doing today (not good, due to Garnett): https://frontend.gutools.co.uk/commercial/kpi/revenue